package.json

```diff
@@ -75,6 +77,7 @@
     "react-virtual": "^2.3.2",
     "resize-observer-polyfill": "^1.5.1",
     "sat": "^0.8.0",
+    "scheduler": "^0.19.1",
```

Comment:
Ditto here, this needs to be removed

Reply:
Ditto here. https://github.com/google/web-stories-wp/runs/1839872700#step:10:329

Will not build without them